node/index.js

```diff
@@ -0,0 +1,52 @@
+const { Pool } = require('pg')
+
+let pool
+
+const initTable = async (p) => {
+  const client = await p.connect()
+  console.log('Initializing table...')
+  try {
+    const tableInitialized = await client.query(
+      `CREATE TABLE IF NOT EXISTS accounts (
+          id UUID PRIMARY KEY DEFAULT gen_random_uuid(),
+          balance INT8
+        )`
+    )
+    console.log(tableInitialized)
+  } catch (err) {
+    console.log(err.stack)
+  } finally {
+    client.release()
+  }
+}
+
+const insertAccounts = async (p) => {
+  const client = await p.connect()
+  console.log('Inserting new rows into the accounts table...')
+  const balanceValue = [Math.floor(Math.random() * 1000)]
+  try {
+    const accountsInserted = await client.query(
+      'INSERT INTO accounts (id, balance) VALUES (DEFAULT, $1)',
+      balanceValue
+    )
+    console.log(accountsInserted)
+  } catch (err) {
+    console.log(err.stack)
+  } finally {
+    client.release()
+  }
+}
+
+exports.handler = async (context) => {
+  if (!pool) {
+    const connectionString = process.env.DATABASE_URL
+    pool = new Pool({
+      connectionString,
+      max: 1,
+      maxLifetimeSeconds = 1800
```

Comment:
`s/maxLifetimeSeconds = 1800/maxLifetimeSeconds: 1800`
I'm actually not sure how `maxLifetimeSeconds = 1800` works in this context! But it looks like the `Pool` [constructor](https://node-postgres.com/api/pool) takes a configuration object, e.g. [this example](https://github.com/brianc/node-postgres/blob/4fa7ee891a456168a75695ac026792136f16577f/packages/pg-pool/test/lifetime-timeout.js#L13).

Reply:
I actually [removed this argument from the sample code](https://github.com/cockroachlabs/examples-aws-lambda/pull/5/files#diff-ddf78f02f4a8e0bccaa29f6a7b1c18871b90fcf4f273fba9163c0510e5cc212cL46), since `maxLifetimeSeconds` isn't documented as a [supported constructor parameter](https://node-postgres.com/api/pool). I also snuck that update into the [docs examples too](https://github.com/cockroachdb/docs/pull/13044/files#diff-d8d48c09896e14df14fde112cc61e8ed7c1d454cbe1186add052af081100332cL61).

I [opened an issue](https://github.com/cockroachlabs/examples-aws-lambda/issues/7) to figure out the best way to add it back at a later point in time.